### ISSUE TITLE
JsonSyntaxException when calling writePromoToTextView on a product that costs more than $17,500

### ISSUE THREAD

**Issue by USMCFighter94** [Affirm/affirm-android-sdk#3]
When attempting to call writePromoToTextView and passing a cost value greater than $17,500, the Affirm sdk crashes with error:
```
E/AndroidRuntime: FATAL EXCEPTION: OkHttp Dispatcher
                                                                   com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected a string but was BEGIN_ARRAY at line 1 column 12 path $.field
                                                                       at com.google.gson.Gson.fromJson(Gson.java:900)
                                                                       at com.google.gson.Gson.fromJson(Gson.java:853)
                                                                       at com.google.gson.Gson.fromJson(Gson.java:802)
                                                                       at com.google.gson.Gson.fromJson(Gson.java:774)
                                                                       at com.affirm.affirmsdk.AffirmRequest$1.onResponse(AffirmRequest.java:72)
```

**Comment by hectorups** [comment]
Thank you for reporting. There was a bug deserializing the error response from the server.

**Comment by hectorups** [comment]
should be fixed on 1.0.6
